armeria-backend/zio/src/main/scala/sttp/client3/armeria/zio/zio.scala

```diff
@@ -0,0 +1,66 @@
+package sttp.client3.armeria
+
+import _root_.zio._
+import sttp.capabilities.Effect
+import sttp.capabilities.zio.ZioStreams
+import sttp.client3._
+import sttp.client3.impl.zio.{ExtendEnv, SttpClientStubbingBase}
+
+package object zio {
+
+  // Forked from async-http-client-backend/zio
+  // - Removed WebSocket support
+
+  /** ZIO-environment service definition, which is an SttpBackend. */
+  type SttpClient = SttpClient.Service
```

Comment:
can't we have simply `type SttpClient = SttpBackend[Task, ZioStreams]`? I think the pattern of having `Service` objects is deprecated?

Reply:
Changed per suggestion.

Reply:
@adamw Could you please take another look when you get a chance? Your suggestions have been addressed in the revised PR. Thanks!